### ISSUE TITLE
Node.js 16 deprecated actions

### ISSUE THREAD

**Issue by DabrowskiAdamPG** [PavanMudigonda/html-reporter-github-pages#6]
Hi, any plans to update deprecated Node.js 16 based actions to new versions? I get warning annotation:

![image](https://github.com/PavanMudigonda/html-reporter-github-pages/assets/124149912/060c8e58-7812-428f-84d1-d2fcaa559620)

```
Please update the following actions to use Node.js 20: dev-hanz-ops/install-gh-cli-action@v0.1.0, actions/checkout@v3, peaceiris/actions-gh-pages@v3.9.3. For more information see: https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/.
```

**Comment by AdamOakman** [comment]
https://github.com/PavanMudigonda/html-reporter-github-pages/pull/7

**Comment by PavanMudigondaTR** [comment]
merged #7  in v1.3 

please try and let me know @AdamOakman  @DabrowskiAdamPG

**Comment by DabrowskiAdamPG** [comment]
Looks good! Thanks for quick approval 👍
